vignettes/SKG2020_submission/skg2020_submission.Rmd

```diff
@@ -1,19 +1,23 @@
 ---
 title: "Accessing and analysing the OpenAIRE Research Graph Dump with R"
+author:
+  - name: Najko Jahn 
+    url: https://twitter.com/najkoja
+    affiliation: State and University Library Göttingen
+    affiliation_url: https://www.sub.uni-goettingen.de/
 bibliography: literature.bib
-csl: springer-lecture-notes-in-computer-science.csl
 output:
+  bookdown::html_document2:
+    number_sections: false
+    toc: yes
   pdf_document:
     fig_caption: yes
     keep_tex: yes
     template: lncs.sty
 link-citations: yes
 pkgdown:
   as_is: true
-vignette: >
```

Comment:
these are only necessary for actually vignettes, and since we're only doing articles (= never plan on publishing vignettes to cran), this complexity can happily go.
I think it makes more sense to only publish articles, not vignettes, because vignettes just add more headaches and then there's two URLs where users can find roughly the same content (confusing).
I think the tidyverse team has also stopped publishing actual "vignettes" though everyone still calls them vignettes.
(Articles is what pkgdown calls vignettes which are pkgdown-only).

Reply:
Great, was not aware of it!